src/reanimated2/animation/timing.ts

```diff
@@ -65,6 +65,16 @@ export const withTiming = function (
 ): Animation<TimingAnimation> {
   'worklet';
 
+  if (userConfig?.easing) {
+    const { easing } = userConfig;
+    if (!('__workletHash' in easing)) {
```

Comment:
While this is probably safe here, I really don't think we should be using `in` operator in worklets.